experiments.py

```diff
@@ -0,0 +1,83 @@
+import csv
+from datetime import datetime
+from trainTestReinforcementAlgorithm import *
```

Comment:
nit: Should we match the styling of `snakeRewardFuncs` below with something like `import trainTestReinforcementAlgorithm as trainTestFuncs`?

Reply:
Since this is an experiments file and we call these functions a lot, i think it is easier to leave as is.